app/views/smart_proxies/index.html.erb

```diff
@@ -8,7 +8,13 @@
     <tr>
       <th class="col-md-2"><%= s_("SmartProxy|Name") %></th>
       <th class="col-md-4"><%= s_("SmartProxy|Url") %></th>
```

Comment:
I would remove the URL based on the fact that our installer create the entry with the same name and url.